internal/model/organization.go

```diff
@@ -0,0 +1,25 @@
+package model
+
+import "github.com/volatiletech/null/v8"
+
+type UpdateOrganization struct {
+	Id   int         `json:"id" validate:"required,greater_than_zero"`
+	Name null.String `json:"name"`
+}
+
+type Organization struct {
+	Id   int    `json:"id" boil:"id"`
+	Name string `json:"name" boil:"name"`
+}
```

Comment:
We need more fields here... Check the organizations table, it has a lot more columns.. And one of the biggest implications of this change is you have to add the "created_by", and "last_updated_by" which will require you to also rewrite the GET query to include those fields, and that will involve INNER JOINS to the users table.

Reply:
Another [good reference](https://github.com/dembygenesis/theoverwatchtools.cli/blob/33dd61bccf6819341626cb14f1736f6bf80589f2/internal/persistence/persistors/mysqlstore/store_category.go#L187)

Reply:
The additional fields have been added to the organization struct and the query has now been updated.